### PR TITLE
Fix VisualBERT docs

### PR DESCRIPTION
**Pull request by gchhablani**
# What does this PR do?

This PR fixes VisualBERT docs and adds demo link. Please let me know in case of any issues.

Reviewers @LysandreJik @patil-suraj

### CODE REVIEW

File: docs/source/model_doc/visual_bert.rst [hunk 1]
```diff
@@ -58,7 +58,9 @@ layer, and is expected to be bound by [CLS] and a [SEP] tokens, as in BERT. The
 appropriately for the textual and visual parts.
 
 The :class:`~transformers.BertTokenizer` is used to encode the text. A custom detector/feature extractor must be used
-to get the visual embeddings. For an example on how to generate visual embeddings, see the `colab notebook
+to get the visual embeddings. Refer to the `demo notebook
+<https://github.com/huggingface/transformers/tree/master/examples/research_projects/visual_bert>`__ for an example on
+VisualBERT VQA. For an example on how to generate visual embeddings, see the `colab notebook
```
Comment (patil-suraj) [inline docs/source/model_doc/visual_bert.rst:63]: (nit) Could you maybe format these examples in a list?

File: src/transformers/models/visual_bert/modeling_visual_bert.py [hunk 2]
```diff
@@ -1068,13 +1068,13 @@ def forward(
...
-            ... visual_token_type_ids=visual_token_type_ids,
-            ... labels=labels
-            ... }})
+            >>> inputs_dict = {k: v.unsqueeze(0) for k,v in encoding.items()}
+            >>> inputs_dict.update({
+            ...     "visual_embeds": visual_embeds,
+            ...     "visual_attention_mask": visual_attention_mask,
+            ...     "visual_token_type_ids": visual_token_type_ids,
+            ...     "labels": labels
+            ... })
```
Comment (patil-suraj) [inline src/transformers/models/visual_bert/modeling_visual_bert.py:1077]: good catch!

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for fixing this!

File: docs/source/model_doc/visual_bert.rst (hunk 1 above)
Comment (gchhablani) [reply docs/source/model_doc/visual_bert.rst]: Changed ^_^

**Pull request merged**